### PR TITLE
CS5356 Homework

### PR DESCRIPTION
**Pull request by redenter**
Hi,
I've added my details to the github repository. 
Please accept.

### CODE REVIEW

File: people/ashwin-ramanathan.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Ashwin Ramanathan
+--------------
+
+![](photos/ashwin.jpg)
```
Comment (dblock) [inline people/ashwin-ramanathan.md]: Please rename `ashwin.jpg` to `ashwin-ramanathan.jpg`, there might be other Ashwin's.

**Comment by dblock** [comment]
See my comment above + please _squash_ your commits into 1.

**Comment by redenter** [comment]
Hi,
I've squashed the commits into one and have renamed the image file as well.

**Comment by dblock** [comment]
Merged via https://github.com/Cornell-CS5356-Fall2015/cs5356/commit/33090b9a51cdbb6f08375950fb9f2b4ad13a72e6.

**Pull request closed without merging**